test/src/tests-persistence.ts

```diff
@@ -1695,49 +1747,62 @@ describe('migrations and persistence-related', function() {
         });
 
         setCookie(workspaceCookieName, cookies);
-        mParticle.useCookieStorage = true;
+        // FIXME: Should this be in configs or global?
+        window.mParticle.config.useCookieStorage = true;
 
-        mParticle.init(apiKey, window.mParticle.config);
+        window.mParticle.init(apiKey, window.mParticle.config);
 
-        mParticle.getInstance()._Persistence.setFirstSeenTime('current', 10000);
-        var currentFirstSeenTime = mParticle
+        window.mParticle
+            .getInstance()
+            ._Persistence.setFirstSeenTime('current', 10000);
+        const currentFirstSeenTime = window.mParticle
             .getInstance()
             ._Persistence.getFirstSeenTime('current');
-        mParticle.getInstance()._Persistence.setFirstSeenTime('current', 2);
-        mParticle
+        window.mParticle
+            .getInstance()
+            ._Persistence.setFirstSeenTime('current', 2);
+        window.mParticle
             .getInstance()
             ._Persistence.getFirstSeenTime('current')
             .should.equal(currentFirstSeenTime);
 
-        mParticle.getInstance()._Persistence.setFirstSeenTime('previous', 10);
-        mParticle
+        window.mParticle
+            .getInstance()
+            ._Persistence.setFirstSeenTime('previous', 10);
+        window.mParticle
             .getInstance()
             ._Persistence.getFirstSeenTime('previous')
             .should.equal(10);
-        mParticle.getInstance()._Persistence.setFirstSeenTime('previous', 20);
-        mParticle
+        window.mParticle
+            .getInstance()
+            ._Persistence.setFirstSeenTime('previous', 20);
+        window.mParticle
             .getInstance()
             ._Persistence.getFirstSeenTime('previous')
             .should.equal(10);
 
-        mParticle
+        window.mParticle
             .getInstance()
             ._Persistence.getFirstSeenTime('previous_set')
             .should.equal(100);
-        mParticle
+        window.mParticle
             .getInstance()
             ._Persistence.setFirstSeenTime('previous_set', 200);
-        mParticle
+        window.mParticle
             .getInstance()
             ._Persistence.getFirstSeenTime('previous_set')
             .should.equal(100);
         done();
     });
 
-    it('should properly set setLastSeenTime()', function(done) {
+    it('should properly set setLastSeenTime()', done => {
+        debugger;
+
+        const mParticle = window.mParticle;
```

Comment:
**thought** why do this in this test and not _all_ the others? (if its not too late) would create a lot less lines of changes

Reply:
Good callout